docs/connecting-data/connecting-to-apis-and-external-data/donorsearch-api.md

```diff
@@ -2,6 +2,91 @@
 description: How to Connect DataDistillr to the DonorSearch API
 ---
 
-# DonorSearch API
+# Connecting to DonorSearch
 
-Details coming soon!
\ No newline at end of file
+## How to Connect DataDistillr to DonorSearch
+To set up a data source connection for DonorSearch, you will need to have:
+
+- A [unique name](#name) for your data source connection to be used in queries
+- The [API Key](#api-key) generated by your DonorSearch account
+
+### Data Source Form
+To locate the DonorSearch form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+  ![Data Source Wizard][image-7]{ width="100%" }
+</figure>
+
+
+
+On the API screen, select DonorSearch from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-8]{ width="100%" }
+</figure>
+
+
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the DonorSearch API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+  ![DonorSearch Form][image-1]{ width="100%" }
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+!!! info "Acceptable Characters Include"
+
+    - lowercase alphanumeric characters
+    - underscores
+
+
+## Endpoints
+The table below shows a list of endpoints available to connect within the DataDistillr application. If you need to connect to an endpoint not listed below, please use the [Custom API](custom-apis.md) Form.
+
+| Endpoint       | Required                           | Optional                                                                               | Description |
+|----------------|------------------------------------|----------------------------------------------------------------------------------------|-------------|
+| `donor_search` | firstName<br>lastName<br>homeState | middleName<br>homeStreetAddress<br>homeCity<br>homeZip<br>Employer<br>Age<br>Telephone |             |
+| `accountStats` |                                    |                                                                                        |             |
+
+
+### Nav Tree
+The endpoints above will display as follows in the nav tree once your API has successfully connected.
+
+<figure markdown>
+  ![DonorSearch Endpoints][image-2]{ width="100%" }
+</figure>
+
+
+## Sample Queries
+The following queries are intended to help you get started, and make life simpler querying within your API.
+
+For the following examples, suppose that my DonorSearch API data source was called `mydonorsearchapi` and I want to query an endpoint. The endpoint goes after the DonorSearch data source name:
+
+!!! example "FROM Clause"
+
+    ```sql
+    FROM `mydonorsearchapi`.`<ENDPOINT>`
+    ```
+
+### donor_search 
+
+```sql 
+SELECT * FROM donor_search.ds
```

Comment:
Can we swap out the first name, last name, and phone number so we're not sharing Monty's personal details :) It's ok if the search doesn't return anything